pkg/arvo/sys/lull.hoon

```diff
@@ -2762,7 +2762,7 @@
             sky=(map spur path-state)
             ken=(jug spar:ames wire)
     ==  ==
-  +$  egg-any  $%([%15 egg])
+  +$  egg-any  $%([%15 egg %16 egg])
```

Comment:
Should be `[[%15 egg] [%16 egg]]`.
#6855 should help ensure `%spore` and `$egg` are kept in sync.